src/atlas/projection/detail/StretchRegular.h

```diff
@@ -46,9 +51,9 @@ class StretchLAM final : public ProjectionImpl {
      }
 
     void checkvalue(const double&, const double&) const;
-    double general_stretch (double&, const bool&, int&, const int&, const int&) const;
+    double general_stretch (double&, const bool&, int, const int, const int) const;
```

Comment:
Passing something by reference makes the argument be of 64bit (for 64bit architectures).
For this reason it was good to pass int (32bit) by value indeed.
You could do the same with the `bool` to be only a 1bit argument.
The first argument is passed as `double&` which indicates it is a "INTENT(INOUT)" variable (Fortran analogy) whereas it should really be a "INTENT(IN)" which you can specify by passing as const-reference `const double&` or by value `const double` as the others. I prefer the latter for consistency with other arguments and because a value (double) or reference are both 64 bit.

```suggestion
    double general_stretch (const double, const bool, int, const int, const int) const;
```
And make the matching change in StretchRegular.cc of course.